packages/notionable/src/hooks/use-portfolio-store.tsx

```diff
@@ -0,0 +1,12 @@
+import { useContext } from 'react';
+import { PortfolioStoreType } from '../stores/portfolio-store';
+import { RootStoreContext } from '../stores/root-store';
+
+export const usePortfolioStore = (): PortfolioStoreType => {
```

Comment:
If these are all coming off the root store, I think you should just put it all in the same file, `use-root-store`